### ISSUE TITLE
Notes from meeting on 2018-07-19

### ISSUE THREAD

**Issue by coderberry** [sustainers/events#9]
### Attendees

* Pia Mancini
* Justin Dorfman
* Elio Qoshi
* Gunner
* Kevin Owocki
* Eric Berry

### Action items

- [x] @piamancini will draft email to prior attendees the question “Hey, mark your calendars.. we are going to London. If you know folks who are in that region, let us know.” - Add to Google Doc.
[Link here](https://docs.google.com/document/d/1hhLLQYz93JCBzfD41_U-fNdXRux-KsZ-uhGV0Yr3Bl0/edit?usp=sharing)
- [ ] @jdorfman will send the email once the google doc is prepared and approved.
- [ ] We are going to use “Sustain” and not “SustainOSS”, and not focus on the location (e.g. Sustain Summit vs. Sustain UK)
- [ ] @piamancini will create a “meta” issue on GitHub
- [ ] @piamancini will check with Google so we can have location (hotels, etc)
- [ ] @jdorfman will follow up on sponsorships (first ask prior sponsors, then reach out to new ones). Sponsorships start at $2,000 and can scale. Does not include ticket.
- [ ] @coderberry will reach out to new sponsors (after Justin gives go-ahead)
 
### Notes

In terms of people we are going to invite, we need to get that list together. Let’s begin by inviting and getting tickets for those who are coming from overseas.

[Pia] will ask attendees from last year “who was not represented” and start building that list. That way we can invite them. 15-20 people.

Advisor vote was just for the goals of sustain. [Pia] will email them herself.

[Gunner] Email full group with “Hey, mark your calendars.. we are going to London. If you know folks who are in that region, let us know.”

[Gunner] Managing London travel from SF is a nightmare. Wondering if there’s a local partner in London time that could be doing the leasing. Where does money come from? Agree to scope of aspiration. How much is left? $11,958 USD. Reimburse people who cannot front ticket. Next step is to start reaching out to sponsors.  Justin will take lead on sponsorships.

[Pia] Maybe invite Guido van Rossum? His current experiences with leaving Python is valuable to the community.

[Justin] We could also ask Betsy.

[Pia] Maybe look at who is attending Moz Fest and Linux Foundation - Open Source Summit and see if they would like to stay around for Sustain.

[Gunner] Next week, discuss MS acquisition of GitHub. Also, be aware that we are having the convo at Google.

#### Values / Goals doc

Advisory board didn’t give a lot of feedback. They agree it looks fair and gives it the thumbs up.

It’s a good place to start.

Values / Goals Doc: https://docs.google.com/document/d/1YJ-3-Q99oO9b-CUUqqUEyWLlmcWsRQe6WbDD3x7-25w/edit?ts=5b2bc69c

Revised version: https://docs.google.com/document/d/1YjqK-46M25czK5l6iygYYCrqPs4UGEFbaJAKgHsnEJ0/edit#heading=h.spb1evgkon1n

#### Branding (Elio)

Propose to cut out OSS of “SustainOSS”. People aren’t using it anyway and everyone uses “Sustain”. It is unnecessary. Event though it’s a generic word, it works well in our community. Example: “View Source”.

Event would become “Sustain Summit”. This would allow for independent events. Allows focus on content and not location. Location is just a logistical detail, not the heart of sustain.

Everyone agrees.

**Comment by jdorfman** [comment]
This is great, thank you @coderberry!

**Comment by piamancini** [comment]
Here's the link for save the date
https://docs.google.com/document/d/1hhLLQYz93JCBzfD41_U-fNdXRux-KsZ-uhGV0Yr3Bl0/edit?usp=sharing

**Comment by jdorfman** [comment]
@piamancini sent you a preview

`@everyone` preview:

![cloudapp annotation 2018-07-26 at 9 49 55 am png](https://user-images.githubusercontent.com/398230/43276406-6c8f7640-90b9-11e8-824d-8b4245ba9847.png)

**Comment by piamancini** [comment]
Maybe remove the ‘about sustaining open source’ that I added originally? Since they were there last year they know what it’s about and the sentence is a bit wordy.

**Comment by jdorfman** [comment]
@piamancini ok removed, please check your inbox for the latest preview and LMK if it is good to go or not.

**Comment by piamancini** [comment]
Ship it @jdorfman. 👌🏽

**Comment by jdorfman** [comment]
@piamancini done

![cloudapp annotation 2018-07-27 at 7 41 04 am png](https://user-images.githubusercontent.com/398230/43327494-71690b6a-9170-11e8-9f72-bdcf5adcb915.png)
